src/components/teasers/block-teaser.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import styled from 'styled-components';
+import { up } from '../breakpoint/breakpoint';
+
+const Teaser = styled.div``;
+
+const LeftContainer = styled.div<{ splitView: boolean }>`
+  ${up('md')} {
+    text-align: ${(props) => (props.splitView ? 'right' : 'left')};
+    padding-right: ${(props) => (props.splitView ? '60%' : '0')};
+  }
+`;
+
+const RightContainer = styled.div<{ splitView: boolean }>`
+  ${up('md')} {
+    padding-left: ${(props) => (props.splitView ? '40%' : '0')};
+  }
+`;
+
```

Comment:
in figma there's a 24px gap between the left and right container:
<img width="441" alt="Screenshot 2020-05-08 at 13 45 14" src="https://user-images.githubusercontent.com/31735645/81402813-63fae200-9132-11ea-8899-55db180ae60a.png">

Reply:
ups, forgot it. will add it :)